Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
```

Comment:
>__ABContactOperation [](start = 11, length = 20)

You should keep all interfaces in your header file. Especially for internal classes.